spec/dummy/app/models/group.rb

```diff
@@ -0,0 +1,3 @@
+class Group < ActiveRecord::Base
+  has_and_belongs_to_many :users
```

Comment:
should use has_many through
```suggestion
  has_many :users, through: :groups_users
  has_many :groups_users
```